generators/app/templates/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "files.associations": {
+        "dockerfile.*": "dockerfile"
```

Comment:
When I tested on mac the casing of the left side did not matter, but the right side needed to be lowercase so I went lowercase on both sides for consistency.